src/__experimental__/components/switch/switch.component.js

```diff
@@ -124,7 +124,7 @@ Switch.propTypes = {
    */
   size: PropTypes.string,
   /** the value of the checkbox, passed on form submit */
-  value: PropTypes.string.isRequired
+  value: PropTypes.string
```

Comment:
This prop will also need to be updated in the `switch.d.ts` file too.

Reply:
I can see you've addressed this as part of commit 7141372. I'll resolve the conversation for this comment. Thank you.